blocks/collapsible/collapsible.css

```diff
@@ -0,0 +1,87 @@
+:root {
+    /* font color */
+    --mt-collapsible-question-font-color: var(--mt-global-color-base-primary);
+    --mt-collapsible-question-expanded-font-color: var(--mt-global-color-accent-quinary);
+
+    /* font weight */
+    --mt-collapsible-question-hover-font-weight: var(--mt-font-weight-boldest);
+
+    /* line height */
+    --mt-collapsible-question-line-height: 1rem;
+    --mt-collapsible-answer-line-height: 1.5rem;
```

Comment:
Use `--mt-body-line-height: 1.5rem;` resp  `--mt-button-line-height: 1rem;` instead

Reply:
If those values change, it will have a side effect on the collapsible. Not sure it is desirable.